Rakefile

```diff
@@ -119,6 +127,7 @@ task documentation_syntax_check: :yard_for_generate_documentation do
     # TODO: parser cannot parse the example, so skip it.
     #       https://github.com/whitequark/parser/issues/407
     next if cop == RuboCop::Cop::Layout::SpaceAroundKeyword
+    next if %i[RSpec Capybara FactoryBot].include?(cop.department)
```

Comment:
When `documentation_syntax_check` runs after `internal_investigation`, `RuboCop::Cop::Cop.registry` includes `rubocop-rspec`'s cops. So the task should ignore these cops.